queries.sql

```diff
@@ -6,4 +7,56 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempt FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*  second day------ */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
```

Comment:
- [OPTIONAL] There is a small issue here 🤏 `LIKE '%mon%'` means Finds any values that have "mon" in any position, but According to the requirements we need to Find all animals whose name ends with "mon"., So in this case you should use `LIKE '%mon'` 💯

![0](https://user-images.githubusercontent.com/45231276/198494018-5fa5a1f8-be78-4fbe-a998-ed3255ffe238.PNG)

> But for the hard work you've done, your project deserves to be approved, and I am sure that you will fix this minor issue. So I marked it as OPTONAL 👍